roles/backup/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Create ark backup of resources
+  k8s:
+    state : present
+    definition: 
+      apiVersion: ark.heptio.com/v1
+      kind: Backup
+      metadata:
+        name: "{{ backup_name }}"
+        namespace: heptio-ark
+      spec:
+        labelSelector:
+          matchLabels:
+            app: "{{ backup_label }}"
```

Comment:
May be better solution to specify label as a string "app=label", sometimes label key may be different from app. Or just name it "test".